x-pack/plugins/cases/public/components/all_cases/all_cases_list.tsx

```diff
@@ -141,32 +140,23 @@ export const AllCasesList = React.memo<AllCasesListProps>(
             perPage: page.size,
           };
         }
-        setQueryParams(newQueryParams);
+        setUrlQueryParams(newQueryParams);
         deselectCases();
       },
-      [queryParams, deselectCases, setQueryParams]
+      [queryParams, deselectCases, setUrlQueryParams]
     );
 
     const onFilterChangedCallback = useCallback(
       (newFilterOptions: Partial<FilterOptions>) => {
         if (newFilterOptions.status && newFilterOptions.status === CaseStatuses.closed) {
-          setQueryParams((prevQueryParams) => ({
-            ...prevQueryParams,
-            sortField: SortFieldCase.closedAt,
-          }));
+          setUrlQueryParams({ sortField: SortFieldCase.closedAt });
```

Comment:
Just double checking, but we don't need the `prevQueryParams` anymore because the hook grabs them from the url directly right?

Reply:
Exactly!

And if somehow there is nothing in the URL, the local storage also has the values.